cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -24,10 +24,15 @@
 import junit.framework.Assert;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 public class TaskHelperTest {
 
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
 	@Before
 	public void setUp() throws Exception {
```

Comment:
You may wish to use tabs instead of 4xSpaces as would look better surrounding their code

Reply:
Done